packages/apps-v5/src/commands/ps/scale.js

```diff
@@ -12,7 +12,37 @@ https://devcenter.heroku.com/articles/procfile`)
 async function run(context, heroku) {
   let app = context.app
 
+  // will remove this flag once we have
+  // successfully launched larger dyno sizes
+  let isLargerDyno = false
+  const feature = await heroku.get('/account/features/frontend-larger-dynos')
```

Comment:
Could you rename this something like "largerDynoFeatureFlag"? Just in case this lives longer than intended, so it's clear what this is.